src/styles/variables.css

```diff
@@ -1,65 +1,99 @@
-  --grey-blue-2: rgb(244, 246, 250);
-  --grey-blue-3: rgb(246, 246, 246);
-  --action-secondary: rgb(76, 93, 255);
+  --text-disabled: rgb(194, 204, 204);
+  --colony-light-blue: rgb(238, 242, 245);
+  --drop-shadow: rgba(62, 118, 244, 0.14);
 
-  /*
-   * We really need to cleanup the greys, as they're growing larger and larger by the day
-   * Maybe get a count of all the colors from product and try to go from there ...
-   */
-  --temp-grey-0: rgb(195, 204, 205);
-  --temp-grey-1: rgb(188, 197, 216);
-  --temp-grey-2: rgb(46, 50, 53);
-  --temp-grey-4: rgb(153, 153, 153);
+  /* Remane `--grey-1` to `--dark-grey` */
+  --dark-grey: rgb(58, 58, 58);
+
+  /* Remane `--grey-4` to `--grey` */
+  --grey: rgb(124, 124, 124);
+
+  /* Remane `--grey-blue-3` to `--grey-navbar`, potentially to be removed as used only for navbar */
+  --grey-navbar: rgb(246, 246, 246);
+
+  /* Remane `--temp-grey-blue-5` to `--light-grey-pink` */
+  --light-grey-pink: rgb(244, 238, 242); /* Used for user profile gradient start color */
+
+  /* Remane `--temp-grey-blue-6` to `--light-ash` */
+  --light-ash: rgb(231, 242, 246); /* Used for user profile gradient end color */
+
+  /* Remane `--temp-grey-blue-4` to `--light-purple` */
+  --light-purple: rgb(200, 214, 247); /* Used for some box shadows */
+
+  /* Remane `--temp-grey-blue-7` to `--grey-purple` */
+  --grey-purple: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
+
+  /* Remane `--temp-grey-blue-1` to `--light-grey-blue` */
+  --light-grey-blue: rgb(188, 197, 216);
+
+  /* Potential to be renamed */
   --temp-grey-5: rgb(213, 213, 213);
-  --temp-grey-6: rgb(220, 226, 227);
-  --temp-grey-8: rgb(52, 55, 56);
-  --temp-grey-11: rgb(136, 136, 136); /* Used for the text color of time for the 'CardActivity' items */
-  --temp-grey-12: rgb(152, 164, 193); /* Used for Drag & Drop placeholder */
   --temp-grey-13: rgb(227, 224, 227); /* Used for separator background */
-  --temp-grey-blue-1: rgb(242, 246, 246); /* Used for the background-color of comments ('CardActivity' comment items) */
-  --temp-grey-blue-2: rgb(209, 214, 215); /* Used for the border of attachments preview ('CardActivity' attachment items) */
-  --temp-grey-blue-3: rgb(251, 252, 253); /* Used for the board list item ('BordListItem' hover clickable) */
-  --temp-grey-blue-4: rgb(200, 214, 247); /* Used for some box shadows */
-  --temp-grey-blue-5: rgb(244, 238, 242); /* Used for user profile gradient start color */
-  --temp-grey-blue-6: rgb(231, 242, 246); /* Used for user profile gradient end color */
-  --temp-grey-blue-7: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
-  --temp-grey-blue-8: rgb(238, 242, 245); /* Used for the actions list hover background */
-  --drop-shadow: rgba(62, 118, 244, 0.14);
-  --text-disabled: rgb(194, 204, 204);
+  --grey-blue-0: rgb(232, 232, 232);
+  --grey-blue-1: rgb(232, 236, 245);
+  --grey-blue-2: rgb(244, 246, 250);
```

Comment:
These are very close together, and mostly used for backgrounds,

Please combine them into by square root averaging them, which will give you `rgb(236, 238, 242)`

This will look like this:
![Screenshot from 2022-10-31 13-23-49](https://user-images.githubusercontent.com/1193222/198997418-da9a1f5b-fbd0-4379-ad2b-c0b08551d18f.png)

Also, if you need to average more colors, I wrote a quick function to do so:
```js
const quickDirtyRGBAverage = (rgbMatrix) => { // [[232, 232, 232], [232, 236, 245], [244, 246, 250]]
    let r = 0;
    let g = 0;
    let b = 0;
 
    for (let index = 0; index < rgbMatrix.length; index += 1) {
      let [cR, cG, cB] = rgbMatrix[index];
      r += Math.pow(cR, 2);
      g += Math.pow(cG, 2);
      b += Math.pow(cB, 2);
    }

    return [
        Math.round(Math.sqrt((r/rgbMatrix.length))),
        Math.round(Math.sqrt((g/rgbMatrix.length))),
        Math.round(Math.sqrt((b/rgbMatrix.length))),
    ];
} 
```

Reply:
@rdig, thank you for the idea - all implemented. And since there were only 2 colours left with "not good" names I manage to squeeze a bit of creativity and name them properly. Please, let me know if all the changes make sense for you.